index.html

```diff
@@ -70,6 +70,189 @@ <h1 class="primary">
+                </ul>
+              </nav>
+              <button class="action" type="submit">See Project</button>
+            </div>
+          </div>
+          <div class="card">
+            <div class="text">
+              <h2 class="project-title">Multi-Post Stories Gain+Glory</h2>
+              <nav class="tags">
+                <ul>
+                  <li><a href="#">Ruby on rails</a></li>
+                  <li><a href="#">css</a></li>
+                  <li><a href="#">JavScript</a></li>
+                  <li><a href="#">html</a></li>
+                </ul>
+              </nav>
+              <button class="action" type="submit">See Project</button>
+            </div>
+          </div>
+          <div class="card">
+            <div class="text">
+              <h2 class="project-title">Multi-Post Stories Gain+Glory</h2>
+              <nav class="tags">
+                <ul>
+                  <li><a href="#">Ruby on rails</a></li>
+                  <li><a href="#">css</a></li>
+                  <li><a href="#">JavScript</a></li>
+                  <li><a href="#">html</a></li>
+                </ul>
+              </nav>
+              <button class="action" type="submit">See Project</button>
+            </div>
+          </div>
+          <div class="card">
+            <div class="text">
+              <h2 class="project-title">Multi-Post Stories Gain+Glory</h2>
+              <nav class="tags">
+                <ul>
+                  <li><a href="#">Ruby on rails</a></li>
+                  <li><a href="#">css</a></li>
+                  <li><a href="#">JavScript</a></li>
+                  <li><a href="#">html</a></li>
+                </ul>
+              </nav>
+              <button class="action" type="submit">See Project</button>
+            </div>
+          </div>
+          <div class="card">
+            <div class="text">
+              <h2 class="project-title">Multi-Post Stories Gain+Glory</h2>
+              <nav class="tags">
+                <ul>
+                  <li><a href="#">Ruby on rails</a></li>
+                  <li><a href="#">css</a></li>
+                  <li><a href="#">JavScript</a></li>
+                  <li><a href="#">html</a></li>
+                </ul>
+              </nav>
+              <button class="action" type="submit">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      -->
+      <section class="about-me">
+        <div class="text">
+          <span class="top-background"></span>
+          <h2 class="primary">About me</h2>
+          <p class="supporting-text">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+          <button class="action1" type="submit">Get My Resume</button>
+        </div>
```

Comment:
- [ ] [OPTIONAL] In order for us to have a good user experience, kindly add some margin between the two background images in this `div` and the text as the image is currently obstructing some part of the paragraph.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![6](https://user-images.githubusercontent.com/58692590/153648341-cf7ff2f8-b8d0-4816-9f13-c8e444165552.png) | ![66](https://user-images.githubusercontent.com/58692590/153648365-a28ce70b-69f4-4d1c-a902-9a0b5d8061cf.png)